server/app/controllers/ShippingController.ts

```diff
@@ -45,7 +46,7 @@ export class ShippingController extends BaseController {
    * @param request
    * @returns       Shipping Json Format
    */
-  @httpPost('/', TYPES.LoggerMiddleware)
+  @httpPost('/', TYPES.LoggerMiddleware, wrappedCheckRole(['Manufacturing', 'General']))
```

Comment:
I think you should put all roles instead of `Manufacturing `and `General` because this controller is also used in the order/sales pages when creating a new order/sale. If I have `Finance` permission, I can access the order page, but with role restriction, I cannot place a shipment on the order page

Reply:
does creating a sale/order need to use the shipping post endpoint?

Reply:
@tbksujan Yup they do 

https://github.com/Mackbellemore/soen-390-team07/blob/4d7ee297aa5bcb0bb095eed37e3ddd04884ac092/client/src/hooks/useOrderForm.jsx#L67

https://github.com/Mackbellemore/soen-390-team07/blob/4d7ee297aa5bcb0bb095eed37e3ddd04884ac092/client/src/hooks/useSales.jsx#L115

Reply:
> I think that's the point, they are allowed to view but not place any orders/sales because they are not manufacturing. What do you think?

From the sales controller, we can place a [sale](https://github.com/Mackbellemore/soen-390-team07/pull/90/files#diff-f24998eb2889d86868f2e8df74906cedbe9a8b9a21f4bdc5f481db14b5d73284R41) using `Finance `permission, but we cannot create a shipment which will throw an error when calling the shipping controller

Reply:
Try to place an order using `Finance `permission, you will be able to create the order, but not be able to create the shipment

Reply:
ohhhh yeah i see, i'll add all roles to shipping